Compile errors

When I compile my own project I get these errors:
```shell
../react-native-in-app-testing/index.tsx(1,8): error TS1192: Module '"/git/super-project/node_modules/@types/react/index"' has no default export.
../react-native-in-app-testing/index.tsx(217,48): error TS2461: Type 'Set<any>' is not an array type.
```
and build fails.

--- a/index.js
+++ b/index.js
@@ -6,7 +6,7 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
         step((generator = generator.apply(thisArg, _arguments || [])).next());
     });
 };
-import React, { Component } from 'react';
+import * as React from 'react';
 import { ScrollView, Text, View } from 'react-native';
 let descriptions = [];
 let nativeFunctionsAmount = 0;
@@ -17,7 +17,7 @@ function wrapOriginalFunction(originalFunction, functionName) {
         return originalFunction.apply(this, arguments);
     };
 }
-export default class ReactNativeInAppTesting extends Component {
+export default class ReactNativeInAppTesting extends React.Component {
     static it(description, testFunction) {
         descriptions.push(description);
         return testFunction;
@@ -121,8 +121,8 @@ export default class ReactNativeInAppTesting extends Component {
     calculateCoverage() {
         let formattedCoveragePercents;
         if (nativeFunctionsAmount) {
-            const uniqCalledFunctions = [...new Set(calledFunctions)];
-            const coveragePercents = uniqCalledFunctions.length * 100 / nativeFunctionsAmount;
+            const uniqCalledFunctionsCount = new Set(calledFunctions).size;
+            const coveragePercents = uniqCalledFunctionsCount * 100 / nativeFunctionsAmount;
             formattedCoveragePercents = coveragePercents.toFixed(2);
         }
         else {

--- a/index.tsx
+++ b/index.tsx
@@ -1,4 +1,4 @@
-import React, { Component } from 'react'
+import * as React from 'react'
 import { ScrollView, Text, View } from 'react-native'
 
 let descriptions: string[] = []
@@ -40,7 +40,7 @@ interface IProps {
   coverageText: string
 }
 
-export default class ReactNativeInAppTesting extends Component<IProps, IState> {
+export default class ReactNativeInAppTesting extends React.Component<IProps, IState> {
 
   /**
    * @static it - For import to the usage as a test cases wrapper
@@ -214,8 +214,8 @@ export default class ReactNativeInAppTesting extends Component<IProps, IState> {
     let formattedCoveragePercents: string
 
     if ( nativeFunctionsAmount ) {
-      const uniqCalledFunctions: any[] = [ ... new Set( calledFunctions ) ]
-      const coveragePercents: number = uniqCalledFunctions.length * 100 / nativeFunctionsAmount
+      const uniqCalledFunctionsCount: number = new Set( calledFunctions ).size
+      const coveragePercents: number = uniqCalledFunctionsCount * 100 / nativeFunctionsAmount
       formattedCoveragePercents = coveragePercents.toFixed( 2 )
     } else {
       formattedCoveragePercents = '0'

